jwt/api_jws.py

```diff
@@ -90,9 +90,6 @@ def encode(
         if headers and "alg" in headers and headers["alg"]:
             algorithm = headers["alg"]
 
-        if algorithm not in self._valid_algs:
```

Comment:
may I know why this was removed?

Reply:
Because it just `pass` statement, it has no influence except time spent for if condition check